File: engine/BusinessLayer/DTOs/EmergencyEvent/EmergencyEventMarkerDto.cs
using DataAccessLayer.Entities;

namespace BusinessLayer.DTOs.EmergencyEvent;

public class EmergencyEventMarkerDto
{
    public Guid Id { get; set; }
    public Severity Severity { get; set; }
    public EmergencyType Type { get; set; }
    public Status Status { get; set; }
    public decimal Latitude { get; set; }
    public decimal Longitude { get; set; }
}

File: frontend/src/app/interfaces/emergency/EmergencyEventMarkerDto.ts
import { EmergencyType } from './EmergencyType';
import { Severity } from './Severity';
import { Status } from './Status';

export interface EmergencyEventMarkerDto {
  id: string;
  severity: Severity;
  type: EmergencyType;
  status: Status;
  latitude: number;
  longitude: number;
  updatedAt: Date;
}
